pkg/controller/directimagemigration/directimagemigration_controller.go

```diff
@@ -71,8 +71,8 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	// Watch for changes to MigClusters referenced by DirectImageMigrations
 	err = c.Watch(
-		&source.Kind{Type: &migapi.MigCluster{}},
-		handler.EnqueueRequestsFromMapFunc(func(a client.Object) []reconcile.Request {
+		source.Kind(mgr.GetCache(), &migapi.MigCluster{}),
+		handler.EnqueueRequestsFromMapFunc(func(ctx context.Context, a client.Object) []reconcile.Request {
```

Comment:
`ctx context.Context` is not used in the function at all, I did `_ context.Context` instead. My linter complains otherwise.

Reply:
Done